src/pages/items/productId/ProductId.jsx

```diff
@@ -0,0 +1,109 @@
+import Header from "components/Header";
+import BackButton from "components/items/productId/BackButton";
+import Inquire from "components/items/productId/Inquire";
+import InquireItem from "components/items/productId/InquireItem";
+import ProductImage from "components/items/productId/ProductImage";
+import ProductSummary from "components/items/productId/ProductSummary";
+import { useProductIdState } from "components/items/productId/useProductIdState";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import { fetchData } from "api/fetchData";
+import { API_USEDS_PRODUCT_INFORMATION, API_USER_COMMENTS } from "config/api";
+import styles from './ProductId.module.css';
+import Line from "components/items/productId/Line";
+import { ReactComponent as NoInquire } from 'assets/imgs/no_inquire.svg';
+
+const ProductId = () => {
+  const {
+    productName,
+    productPrice,
+    productDescription,
+    productImage,
+    productTags,
+    productFavoriteCount,
+    commentData,
+    updateAt,
+    ownerIamge,
+    ownerNickname,
+    setProductName,
+    setProductPrice,
+    setProductDescription,
+    setProductImage,
+    setProductTags,
+    setProductFavoriteCount,
+    setCommentCount,
+    setCommentData,
+  } = useProductIdState();
+
+  const location = useLocation();
+  const id = location.state.product.id;
+
+  useEffect(() => {
+    const asyncFunction = async () => {
+      const response = await fetchData(`${API_USEDS_PRODUCT_INFORMATION}/${id}`, {productId: id});
+      const product = response.data;
+      setProductName(product.name);
+      setProductPrice(product.price);
+      setProductDescription(product.description);
+      setProductImage(product.images);
+      setProductTags(product.tags);
+      setProductFavoriteCount(product.favoriteCount);
```

Comment:
P2:
각 데이터의 항목을 다 state로 쪼개 저장하니 코드를 파악하기 어려운 것 같습니다.
하나의 state로 관리하시기를 추천드려요.

```suggestion
      setProductData(product);
```